core/config/envvar/schema.go

```diff
@@ -291,6 +293,7 @@ func DefaultValue(name string) (string, bool) {
 	return "", false
 }
 
+// ZeroValue TODO
```

Comment:
Need context for the comment

Reply:
```suggestion
// ZeroValue returns the zero value for a named field, or panics if it does not exist.
```

Reply:
Sorry, I had force pushed before seeing this so the commit was invalid when I tried to add your suggestion.